src/base58.cpp

```diff
@@ -273,6 +273,38 @@ CTxDestination CRavenAddress::Get() const
         return CNoDestination();
 }
 
+bool CRavenAddress::GetIndexKey(uint160& hashBytes, int& type) const
```

Comment:
we could skip the revert if these 3 functions were rewritten to leverage (#1e46ebd Implement {Encode,Decode}Destination without CBitcoinAddress)

Reply:
this comment shouldnt be outdated. wtf github.